icons/donut.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="10" />
+  <circle cx="12" cy="12" r="3" />
```

Comment:
This donut needs, sprinkles, a bite mark or at least a glaze.

![](https://github.com/lucide-icons/lucide/assets/25524993/c763314c-4642-4bee-bfa3-94e9f5434699)

Reply:
@jguddas Something like these…?

![preview](https://github.com/lucide-icons/lucide/assets/7797479/5cef0e13-29c6-46ed-8419-c25ed7dc9452)

Reply:
Yes, I think 1/4 looks the best.

Reply:
Yeah I’m good with either. Need _another_ vote…